### PR TITLE
Feature CI/CD upload artifacts to s3 bucket

### PR DESCRIPTION
**Pull request by yilmazbahadir**
* Upload android artifacts to s3 bucket (s3://symbol-mobile-builds)
   * note that there is a s3 bucket retention policy deleting the files older than 60 days 
   * apk artifacts will be available at https://symbol-mobile-builds.s3.us-west-2.amazonaws.com/list.html
image

* IOS deployment script is reformatted

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -90,6 +90,19 @@ pipeline {
             }
         }
 
+        stage('Upload Android') {
+            when {
+                expression {
+                    TARGET_OS == 'All' || TARGET_OS == 'Android'
```
Comment (Jaguar0625) [inline Jenkinsfile:96]: why All ?

File: Jenkinsfile [hunk 2]
```diff
@@ -90,6 +90,19 @@ pipeline {
...
+                }
+            }
+            steps {
+                // upload apk artifact to folder(env.branch_name) in AWS S3 bucket (s3://symbol-mobile-builds/branches) 
+                // note that there is a s3 bucket retention policy deleting the files older than 60 days
```
Comment (Jaguar0625) [inline Jenkinsfile:101]: isn't 60 days a bit long?

File: Jenkinsfile (hunk 1 above)
Comment (yilmazbahadir) [reply Jenkinsfile:96]: All == IOS + Android

From manual build parameters page:
image

File: Jenkinsfile (hunk 2 above)
Comment (yilmazbahadir) [reply Jenkinsfile:101]: Previously checked with @cryptoBeliever, he said we might need the files for 2-3 months.

Comment (Jaguar0625) [reply Jenkinsfile:101]: are these published (release) builds or development builds?

Comment (yilmazbahadir) [reply Jenkinsfile:101]: for feature branches: dev/testing purposes
dev/main branches: release candidates

btw the release is done manually(supervised).

Comment (yilmazbahadir) [reply Jenkinsfile:101]: Asked @cryptoBeliever, he thinks that we can reduce it to 4 weeks (or 3 weeks if we push it ), good enough?

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**